src/app/admin-view/admin-forms/create-common-project/create-common-project.component.ts

```diff
@@ -0,0 +1,64 @@
+import {Component, OnInit} from '@angular/core';
+import {AbstractControl, FormBuilder, FormGroup} from '@angular/forms';
+import {HttpClient} from '@angular/common/http';
+import {CreateCommonProjectService} from '../services/create-common-project.service';
+import {
+  FormEnumerationData,
+  FormField,
+  FormFileData,
+  FormMultichoiceData,
+  FormRowData
+} from '../../../components/form-fields/models/form-data';
+import formData from '../resources/create-project-data.json';
+
+
+@Component({
+  selector: 'app-create-project',
+  templateUrl: './create-common-project.component.html',
+  styleUrls: ['./create-common-project.component.scss']
+})
+export class CreateCommonProjectComponent implements OnInit {
```

Comment:
V tomto komponte by sme mohli použiť loading spinner keď sa bude vytvárať projet aby to náhoodu neklikali. Môžeš sa inšpirovať login formom alebo contact formom.